packages/esm-patient-programs-app/src/programs/programs-form.workspace.tsx

```diff
@@ -258,7 +247,7 @@ const ProgramsForm: React.FC<ProgramsFormProps> = ({
       value: completionDate,
     },
     {
-      style: { width: '50%' },
+      style: { width: '70%' },
```

Comment:
Let it take the full width.
```suggestion
      style: { width: '100%' },
```